Signal/src/Jobs/MessageFetcherJob.swift

```diff
@@ -38,7 +38,7 @@ public class MessageFetcherJob: NSObject {
 
         Logger.info("\(self.logTag) fetching messages via REST.")
 
-        let promise = self.fetchUndeliveredMessages().then { (envelopes: [SSKEnvelope], more: Bool) -> Promise<Void> in
+        let promise = self.fetchUndeliveredMessages().then { (envelopes: [SSKProtoEnvelope], more: Bool) -> Promise<Void> in
```

Comment:
In this PR, I've generated wrapper for all of the "SSK" protos.  In a follow-up PR, I'll generate for our 4 other schemas as well.

`SSKProtoEnvelope` is the generated replacement for `SSKEnvelope`.

Reply:
Note that I'm using the prefix `SSKProto` instead of `SSK`.  The intention is to avoid namespace collisions and improve readability.